src/components/navbar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+  <div>
+    <nav>
+      <ul>
+        <li><Link to="/">Home</Link></li>
+        <li><Link to="/calculator">Calculator</Link></li>
+        <li><Link to="/quote">Quote</Link></li>
+      </ul>
+    </nav>
+  </div>
```

Comment:
- [ ] **[OPTIONAL]** I think that the wrapping div isn't needed here, since you're using this `<Navbar />` component inside another one, you end up with many wrapping `<div>` elements that make your page structure hard to read! Please consider removing it! 🤓 🔧 

```suggestion
  
    <nav>
      <ul>
        <li><Link to="/">Home</Link></li>
        <li><Link to="/calculator">Calculator</Link></li>
        <li><Link to="/quote">Quote</Link></li>
      </ul>
    </nav>
  
```

Reply:
Well noted.